src/core/resources/assets/customNetworkAssets.ts

```diff
@@ -220,7 +220,7 @@ async function customNetworkAssetsFunction({
                   isNativeAsset: true,
                   name: chain.nativeCurrency.symbol,
                   symbol: chain.nativeCurrency.symbol,
-                  uniqueId: `${nativeAssetAddress}_${chain.id}`,
+                  uniqueId: `${'eth'}_${chain.id}`,
```

Comment:
is this 'eth' here right?

Reply:
yeah, uniqueid for native assets is eth otherwise i can't filter these out of the token list and you'll see duplicated native assets for this